Runner.php

```diff
@@ -40,4 +41,19 @@ public function getAdditionalReporters()
     {
         return $this->additionalReporters;
     }
+
+    public function disableAllChecksExcept(array $checkAlias = array())
```

Comment:
You are actually removing aliases and not just disabling them. Please rename this function accordingly (or are you planning to provide a way of enabling them again ?). phpdoc would be nice for this function.